### PR TITLE
Adopt skeleton

### PR DESCRIPTION
**Pull request by JoshuaAppleby**
Adopt python pip skeleton.
Update docs.

### CODE REVIEW

**Comment by JoshuaAppleby** [comment]
Do we want pytac to release on pypi on each tag?

**Comment by aawdls** [comment]
> Do we want pytac to release on pypi on each tag? 
Yes
> Do we want to change master to main after this PR is merged?
Yes

File: src/pytac/__main__.py
Comment (aawdls) [inline src/pytac/__main__.py:1]: Is this file and the associated test useful for us?

File: src/pytac/__init__.py [hunk 1]
```diff
@@ -26,7 +31,12 @@
 """Error 402 is suppressed as we cannot import these modules at the top of the
```
Comment (aawdls) [inline src/pytac/__init__.py:31]: Looks like you have changed the directive this is referring to

**Review by aawdls — APPROVED** [review]
Looks good - with a couple comments

File: src/pytac/__init__.py (hunk 1 above)
Comment (JoshuaAppleby) [reply src/pytac/__init__.py:31]: I'm assuming it is designed to be below the `from . import` statement? Equally, the comment should be updated from E402 to isort:skip?

File: src/pytac/__main__.py
Comment (JoshuaAppleby) [reply src/pytac/__main__.py:1]: Its standard with the skeleton and allows for cli checking of versions, so I'm assuming yes just for consistency.

**Pull request merged**